nse_utility.h

```diff
@@ -8,9 +8,9 @@ class Target;
 #include "nmap_config.h"
 #endif
 
-#if HAVE_STDINT_H
+//#if HAVE_STDINT_H
```

Comment:
It doesn't seem to compile in windows when adding the HAVE_STDINT_H flag in nmap_winconfig.h